requirements.devel.txt

```diff
@@ -23,6 +23,8 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
 # Requirements for testing development version of important packages.
+# -e git+git://github.com/zenodo/zenodo-accessrequests.git#egg=zenodo-accessrequests
```

Comment:
``-e git+https://github.com/zenodo/zenodo-accessrequests.git#egg=zenodo-accessrequests`` and put it to second last line. I know it will fail until the PR has been integrated.